tools/make_update_yml.py

```diff
@@ -0,0 +1,179 @@
+            '4. Wait for the file copy operation to complete.',
+            '5. Power cycle the board. It will now enumerate and mount as `DAPLINK` or the name of the board.'
+        ]),
+        ('linux', [
+            '1. Download the firmware file.',
+            '2. While holding down the boards reset button, connect the boards USB debug port to the computer. It should enumerate as `BOOTLOADER` or `MAINTENANCE`. For boards that enumerate as `BOOTLOADER` [see our blog to determine if an update for the DAPLink bootloader is available.](https://os.mbed.com/blog/entry/DAPLink-bootloader-update/)',
+            '3. In a terminal execute',
+            '   - `cp <path to firmware file> <MAINTENANCE> && sync`',
+            '   - Note: make sure to change `MAINTENANCE` to the name of the mount point of the drive on your system.',
+            '4. Power cycle the board. It will now enumerate and mount as `DAPLINK` or the name of the board.'
+        ])
+    ]),
+    'lpc11u35':InstructionList([
+        ('windows', [
+            '1. Download the firmware file.',
+            '2. While holding down the boards reset button, connect the boards USB debug port to the computer. It should enumerate and mount as `CRP DISABLD`',
+            '3. Delete the file named `firmware.bin`',
+            '4. Wait for the file copy operation to complete.',
+            '5. Power cycle the board. It will now enumerate and mount as `DAPLINK` or the name of the board.'
+        ]),
+        ('osx', [
+            '1. Download the firmware file.',
+            '2. While holding down the boards reset button, connect the boards USB debug port to the computer. It should enumerate and mount as `CRP DISABLD`',
+            '3. Delete the file named `firmware.bin`',
+            '4. In a terminal execute',
+            '   - `sudo mount -u -w -o sync /Volumes/CRP\ DISABLD ; rm /Volumes/CRP\ DISABLD/firmware.bin && cp -X <path to firmware file> /Volumes/CRP\ DISABLD/`',
+            '5. Wait for the file copy operation to complete.',
+            '6. Power cycle the board. It will now enumerate and mount as `DAPLINK` or the name of the board.'
+        ]),
+        ('linux', [
+            '1. Download the firmware file.',
+            '2. While holding down the boards reset button, connect the boards USB debug port to the computer. It should enumerate and mount as `CRP DISABLD`',
+            '3. Delete the file named `firmware.bin`',
+            '4. In a terminal execute',
+            '   - `cp <path to firmware file> <CRP DISABLD> && sync`',
+            '   - Note: make sure to change `CRP DISABLD` to the name of the mount point on your system.',
+            '5. Power cycle the board. It will now enumerate and mount as `DAPLINK` or the name of the board.'
+        ])
+    ])
+
+}
+
+
+def string_writer(element):
```

Comment:
Indent spacing is off from here to the end of the file. Should be 4 spaces per indent.

Reply:
Thank Chris, I have just updated the files